pkg/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -0,0 +1,474 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is a lift of all the different types spread across multiple files and dropped into a single file. It is also adds the  `docgen` annotations.